### DISCUSSION TITLE
How to suppress STL function call stack when debug?

### DISCUSSION THREAD

**Discussion by Mactarvish** [microsoft/vscode-cpptools#12189] [Q&A] [answered]
CMake debugging works fine for me, but there's a tiny problem that when gdb hit break point all STL function calling will show in call stack, and usually it is unnecessary and makes my call stack too complacated.
Any way to suppress it and only keep my code ? (i.e. like python config `justmycode`)

**Comment by gcampbell-msft** [comment]
@Mactarvish Is this issue happening when debugging cmake itself? Or when debugging your c/c++ code?

**Reply by Mactarvish** [reply]
debugging my c/c++ code

**Reply by gcampbell-msft** [reply, marked as answer]
Understood. I think this issue / question would get a better answer from the maintainers of the c/c++ extension. I'll transfer this post (or at least ping the maintainers). Thanks!

**Reply by gcampbell-msft** [reply]
@sean-mcmanus Could you transfer this to cpp-tools?
